xlwt/Bitmap.py

```diff
@@ -260,3 +267,14 @@ def __init__(self, filename):
         self._rec_data = pack("<HHL", cf, env, lcb) + data
 
 
+class ImDataBmpRecord(ImRawDataBmpRecord):
+    def __init__(self, filename):
+        """Insert a 24bit bitmap image in a worksheet. The main record required is
+        IMDATA but it must be proceeded by a OBJ record to define its position.
+
+        """
+        BiffRecord.__init__(self)
+
+        self.width, self.height, self.size, data = _process_bitmap(filename)
+        self._write_imdata(data)
+
```

Comment:
This doesn't feel right. Not sure we need another whole new class. 
Could we just have one class that takes data and change the calling code of the one that uses the filename?